python/py-thinc/Portfile

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                          1.0
+PortGroup           python          1.0
+PortGroup           github          1.0
+
+name                py-thinc
+version             7.0.3
+revision            0
+github.setup        explosion thinc ${version} v
+
+checksums           rmd160  9b44e80fd2a7ce2ccab0c98b27e879dcc6ffa147 \
+                    sha256  58615763d6af89b19fdcbde91ac94574d1e9eac9e48a4b1ca720f42391534c64 \
+                    size    232776
+
+platforms           darwin
+supported_archs     x86_64
+
+license             MIT
+
+maintainers         {jonesc @cjones051073} openmaintainer
+
+description         spaCy's Machine Learning library for NLP in Python
+long_description    ${description}
+
+# Support python versions
+python.versions     27 35 36 37
+
+if {${name} ne ${subport}} {
+
+    depends_build-append \
+        port:py${python.version}-setuptools \
+        port:py${python.version}-hypothesis
+
+    depends_lib-append \
+        port:py${python.version}-cython \
+        port:py${python.version}-cymem \
+        port:py${python.version}-murmurhash \
+        port:py${python.version}-numpy \
+        port:py${python.version}-tqdm \
+        port:py${python.version}-blis \
+        port:py${python.version}-plac \
+        port:py${python.version}-pathlib2 \
```

Comment:
pathlib2 is only needed for Python < 3.4; mock and flake8 are probably only supposed to be test dependencies

Reply:
done